src/helpers/createUdpStream.js

```diff
@@ -0,0 +1,19 @@
+const dgram = require('dgram');
+const { Writable } = require('stream');
+
+const createUdpStream = (endpoint) => {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=javascript%3AS3524)